backend/src/LAN.ts

```diff
@@ -64,6 +68,19 @@ class LAN {
     })
   }
 
+  async checkForWifiNetwork(ssid: string, callback: (b: boolean) => void) {
```

Comment:
This is marked as async, which indicates a Promise, but it's using callbacks. This should return a promise instead of using callbacks. You can reference sudoPromise.ts for how to wrap a callback in a promise.

Reply:
yeah, I copied the function header =(. Anyway, I simplified the logic and updated the new function to be async and use a promise

Reply:
👍